css/main.css

```diff
@@ -346,22 +357,33 @@ ul {
   color: #6070ff;
 }
 
+.form > .submit-button:hover {
+  border-radius: 8px;
+  width: 9rem;
+  margin-left: 10px;
+  height: 50px;
+  background-color: #6070ff;
+  margin-bottom: 15px;
+  font-weight: normal;
+  font-size: 17px;
+  color: white;
+  border: 1px solid white;
+}
```

Comment:
- [ ] KIndly make sure to align the input's content at the stating of the input box as it is on the provided Figma design. :wink: 

   - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/192553052-45530a9c-069d-4c9d-a132-358b58d3f9f0.png)


   - ### Figma design
    
     ![image](https://user-images.githubusercontent.com/87197412/192553615-3603f31e-cc54-43d8-a8b8-41d6719aa8a6.png)